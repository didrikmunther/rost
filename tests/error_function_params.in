fn b(a: pointer<int>) -> int {
	return 1;
}

fn main() {
	b(5);
}