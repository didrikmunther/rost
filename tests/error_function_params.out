  --> ./tests/error_function_params.in:6:4, ./tests/error_function_params.in:1:6 => CompilerError
1 | fn b(a: pointer<int>) -> int {
  |      ^
  |      └─ Function takes parameter of type: *int
2 |     return 1;
    ...
5 | fn main() {
6 |     b(5);
  |       ^
  |       └─ Wrong type in argument: int
7 | }